app/services/analytics.rb

```diff
@@ -124,11 +124,6 @@ def browser_attributes
   # rubocop:disable Layout/LineLength
   ACCOUNT_RESET_VISIT = 'Account deletion and reset visited'
   DOC_AUTH = 'Doc Auth' # visited or submitted is appended
-  IDV_ADDRESS_VISIT = 'IdV: address visited'
-  IDV_ADDRESS_SUBMITTED = 'IdV: address submitted'
-  IDV_BASIC_INFO_VISIT = 'IdV: basic info visited'
-  IDV_BASIC_INFO_SUBMITTED_FORM = 'IdV: basic info form submitted'
-  IDV_BASIC_INFO_SUBMITTED_VENDOR = 'IdV: basic info vendor submitted'
```

Comment:
just confirmed that these 3 events were removed in #3682 and seem to have been unused at that time, so I think there's no need to link these to current events, they're super out of date